cruise-control/src/main/resources/yaml/endpoints/addBroker.yaml

```diff
@@ -36,8 +36,6 @@ AddBrokerEndpoint:
           type: array
           items:
             type: string
-            # default is every goal
-          example: ["RackAwareGoal", "ReplicaCapacityGoal", "ReplicaDistributionGoal"]
```

Comment:
Why removing the examples?

Reply:
When using the generated openapi.json in swagger it fills the request parameters with values that were given in the `example` part of the yaml. So you had to delete them from the UI before you could run a request. (bc they often failed the request)

Reply:
I think it makes more sense to me to leave them there. Users can easily change the value as needed. I think it's helpful to have those examples in each request.